app/src/keymap.c

```diff
@@ -270,8 +270,9 @@ int zmk_keymap_sensor_triggered(uint8_t sensor_number, const struct device *sens
                 continue;
             }
 
-            const uint32_t position = ZMK_VIRTUAL_KEY_POSITION_SENSOR(sensor_number);
-            ret = behavior_sensor_keymap_binding_triggered(binding, sensor, position, timestamp);
+            struct zmk_behavior_binding_event event = {
+                .position = ZMK_VIRTUAL_KEY_POSITION_SENSOR(sensor_number), .timestamp = timestamp};
+            ret = behavior_sensor_keymap_binding_triggered(binding, sensor, event);
```

Comment:
@joelspadin  Thoughts on if we should pass behaviors the actual sensor position, then let them generate the virtual key position if they need to queue behaviors?

Reply:
As mentioned on Discord, my main concern with that is that it would be very easy to just pass the sensor position along when queueing behaviors, which could cause very subtle bugs.